packages/plugins/graphql/server/src/services/builders/resolvers/query.ts

```diff
@@ -8,15 +8,28 @@ export default ({ strapi }: Context) => ({
     const { uid } = contentType;
 
     return {
-      async find(parent: any, args: any, ctx: any) {
+      async findMany(parent: any, args: any, ctx: any) {
         await validate.contentAPI.query(args, contentType, {
           auth: ctx?.state?.auth,
         });
+
+        const sanitizedQuery = await sanitize.contentAPI.query(args, contentType, {
+          auth: ctx?.state?.auth,
+        });
+
+        return strapi.documents!(uid).findMany({ status: 'published', ...sanitizedQuery });
```

Comment:
Is `status` being over-writable made on purpose?

Reply:
yes by default we act on published data but you can want to act on the draft